pipelines/ci-sit-kubernetes-pipeline.yml

```
@@ -570,9 +584,69 @@ jobs:
       docker-image-resource: fieldwork-adapter-docker-latest,
       kubernetes-repo: census-rm-kubernetes-microservices-repo}
 
+- name: "CI Deploy Notify Processor"
+  serial: true
+  serial_groups: [ci-notify-processor]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-microservices-repo
+    trigger: true
+  - get: notify-processor-docker-latest
+    trigger: true
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
+    on_failure: *slack_failure_alert_ci
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((ci-gcp-project-name))
+      KUBERNETES_CLUSTER: ((ci-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: notify-processor
+      KUBERNETES_SELECTOR: app=notify-processor
+      KUBERNETES_FILE_PATH: kubernetes-repo/microservices
+      KUBERNETES_FILE_PREFIX: notify-processor
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: notify-processor-docker-latest,
+      kubernetes-repo: census-rm-kubernetes-microservices-repo}
+  
+- name: "CI Deploy Notify Stub"
+  serial: true
+  serial_groups: [ci-notify-stub]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-microservices-repo
+    trigger: true
+  - get: notify-stub-docker-latest
+    trigger: true
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
```

```suggestion
    file: census-rm-deploy/tasks/kubectl-apply-service-and-deploy.yml
```
Since the notify stub has a service also.